ADALiOS/ADALiOS/ADKeychainTokenCacheStore.m

```diff
@@ -536,6 +554,15 @@ -(void) removeAllForUser: (NSString*) userId
                    error: (ADAuthenticationError* __autoreleasing*) error
 {
     API_ENTRY;
+    if (mSharedCache)
+    {
+        AD_LOG_ERROR(@"******** removeAllWithError is being called. This will remove all tokens from the ADAL shared cache that your application has access to. ********", AD_ERROR_UNEXPECTED, nil);
```

Comment:
It's supposed to be on removeAllWithError: I just copied & pasted into the wrong spot. This removeAllForUser API is a private thing just for the broker. :-/